screens/Camera.js

```diff
@@ -1,149 +1,174 @@
   // to create an Airtable record with the image.
-  _uploadImage = async imageBinary => {
-    const apiUrl = 'https://api.cloudinary.com/v1_1/dxecwnkxx/image/upload';
-    const data = {
-      file: imageBinary,
-      upload_preset: IMG_KEY
-    };
+  // TODO: @Johnathan, move the airtable portion to the other file
+  _uploadImage = async (imageBinary) => {
+    let apiUrl = 'https://api.cloudinary.com/v1_1/dxecwnkxx/image/upload';
+    let data = {
+      "file": imageBinary,
+      "upload_preset": IMG_KEY,
+    }
     fetch(apiUrl, {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json'
       },
       body: JSON.stringify(data)
     })
-      .then(resp => {
-        return resp.json();
-      })
-      .then(resp => {
-        const postUrl = resp.secure_url;
-        return BASE('images').create(
-          [
-            {
-              fields: {
-                Attachments: [
-                  {
-                    url: postUrl
-                  }
-                ]
+    .then(data => {
+      return data.json()
+    }).catch(err => {
+      console.error(err)
+    })
+    .then(data => {
+      let postUrl = data.secure_url
+      let currentDate = new Date()
+      return BASE('Receipts').create([
+        {
+          "fields": {
+            "Transaction": [
+              this.state.transactionId
+            ],
+            "Time": currentDate.toISOString(),
+            "Customer": [
+              this.state.customerId
+            ],
+            "Attachments": [
+              {
+                "url": postUrl
               }
-            }
-          ],
-          { typecast: true }
-        );
+            ]
+          }
+        }
+      ], {typecast: true})
+    }).catch(err => {
+      console.error(err)
+    })
+    .then(() => {
+      this.setState({
+        uploadStatus: "Upload Completed"
       })
-      .then(() => {
-        this.setState({
-          uploadStatus: 'Upload Completed'
-        });
-      });
-  };
+    }).catch(err => {
+      console.error(err)
+    })
+  }
+
+  _returnHome = async () => {
+    this.props.navigation.navigate('Home');
+  }
 
   render() {
     const { hasCameraPermission } = this.state;
-    // Before asking for permissions, renders blank screen
     if (hasCameraPermission === null) {
       return <View />;
-    }
-    if (hasCameraPermission === false) {
+    } else if (hasCameraPermission === false) {
       return <Text>No access to camera</Text>;
-    }
-    return (
-      <View style={{ flex: 1 }}>
-        <Camera
-          style={{ flex: 1 }}
-          type={this.state.type}
-          ref={ref => {
-            this.camera = ref;
-          }}
-          base64>
-          <View
-            style={{
-              flex: 1,
-              backgroundColor: 'transparent',
-              flexDirection: 'row'
-            }}>
-            <TouchableOpacity
+    } else {
+      return (
+        <View style={{ flex: 1 }}>
```

Comment:
Just a reminder that we're all in `styled-components` now -- that can be in the next PR from part 2 tasks though

Reply:
yes